pyexcel/book.py

```diff
@@ -56,10 +56,6 @@ def load_from_sheets(self, sheets):
         if sheets is None:
             return
         keys = sheets.keys()
-        if not isinstance(sheets, compact.OrderedDict):
-            # if the end user does not care about the order
-            # we put alphatical order
-            keys = sorted(keys)
```

Comment:
There might've been a good reason for this, but it's not obvious when I run the tests? 🤷

Reply:
Originally, without the 'project' function(in 0.6.2), it was simply a good wish to sort them if the incoming dict is not an ordered dict. Since python 3.6, the dict is automatically ordered, hence, it makes sense to remove it.

Reply:
on the sequence of book names, I have no objection in removing it. since python 3.6, most of the dict are ordered.